pkg/aspect/clean/clean.go

```diff
@@ -107,9 +102,9 @@ func NewDefault(isInteractive bool) *Clean {
 }
 
 // Run runs the aspect build command.
-func (c *Clean) Run(_ *cobra.Command, _ []string) error {
+func (c *Clean) Run(_ *cobra.Command, _ []string, isInteractiveMode bool) error {
```

Comment:
Since you are here - if we are not using `cmd` and `args`, and we don't plan to use them, let's just remove the args here. The commands don't satisfy any interface to enforce this signature.